js/src/components/detailed-error/index.js

```diff
@@ -12,29 +13,38 @@ import warningIconUrl from '~/images/icons/warning.svg';
 import './index.scss';
 
 const DetailedError = ( { errorSlots, className } ) => {
-	const error = useDetailedErrorBySlots( errorSlots );
+	const errors = useDetailedErrorBySlots( errorSlots );
 
-	if ( ! error ) {
+	if ( ! errors || errors.length === 0 ) {
 		return null;
 	}
 
-	const { title, description } = getFormattedErrorMessage( error );
+	const formattedErrors = getFormattedErrorMessage( errors );
 
 	return (
-		<div className={ classNames( 'gla-detailed-error', className ) }>
-			<div className="gla-detailed-error__title">
-				<span>
-					<img
-						src={ warningIconUrl }
-						alt=""
-						width={ 16 }
-						height={ 16 }
-					/>
-				</span>
-				<h4>{ title }</h4>
-			</div>
-			<p className="gla-detailed-error__description">{ description }</p>
-		</div>
+		<Fragment>
+			{ formattedErrors.map( ( { title, description }, idx ) => (
+				<div
+					key={ idx }
```

Comment:
@ankitrox Is there another property from the `error` object which we can use to uniquely identify that error? Using the index as key is an anti pattern.